jazz_create-serverless-service/components/validate-arn.js

```diff
@@ -0,0 +1,26 @@
+/** 
+  @module: validate-arn.js
+  @description: Validates ARN value for a lambda function in this case Authorizer
+  @author:
+  @version: 1.0
+**/
+
+var validateARN = (arn) => {
+
+  var arnvalues = arn.split(":");
```

Comment:
what if arn is null or undefined. Have you already validated that? Even if you have for all public(exportable) functions you should check if its not null/undefined.
Please add a unit test for this

Reply:
Already validating the arn. Also included validation in the exportable funciton. Unit test case is handling in the separate story. So leaving the unit test case part as of now.